src/test/kv_store_bench.cc

```diff
@@ -420,7 +420,6 @@ int KvStoreBench::test_teuthology_aio(next_gen_t distr,
       break;
     }
 
-    delete cb_args;
```

Comment:
nice catch! could you take this opportunity to add an `assert(false)` to handle the `default` case of switch-case block above?

Reply:
may be use  `ceph_abort_msg("unrecognized operation")` instead ?  Does this commit need an individual commit id?

Reply:
IMHO, no need. it just serves as a comment to help the reader to understand this never happens.